site/en/docs/extensions/mv3/intro/index.md

```diff
@@ -1,51 +1,60 @@
+This site introduces Manifest V3, short for Manifest Version 3, which is the latest iteration of the Chrome extension platform. It shares the background and reasons for introducing Manifest V3 and the vision for the platform's future, along with resources on how to migrate.
 
-# A list of authors. These usernames correspond to the keys in the
-# _data/authorsData.json file.
+## What is a Manifest?
 
----
+An extension [Manifest][doc-manifest] gives the browser information about the extension, such as
+the most important files and the capabilities the extension might use. The extension platform features change when there's a new [Manifest version][manifest-version].
+
+## Introducing Manifest V3 
+
+Manifest V3 represents one of the most significant shifts in the extensions platform since it
+launched a decade ago. Manifest V3 extensions will enjoy enhancements in security, privacy, and
+performance; they can also use more contemporary Open Web technologies such as service workers and
+promises. 
+
+## Manifest V3 resources
+
+Manifest V3 is part of a shift in the philosophy behind how we approach end-user security and privacy. The following articles provide an overview of Manifest V3, the reasons behind it, and how to
+approach it:
+
+[Platform vision][mv3-platform]
+: Explains how the Manifest V3 changes fit into the big picture of where the platform is going.
+
+[Overview of Manifest V3][mv3-overview]
+: Summarizes the technical changes introduced with Manifest V3.
+
+[Migration guide][mv3-migration]
+: Describes how to get started updating Manifest V2 extensions so they work in Manifest V3.
 
-This site introduces Manifest V3 for Chrome Extensions (Manifest V3). It presents the
-background and reasons for introducing Manifest V3 and our vision for the platform's
-future, along with guidance on how to optimize your extensions to use Manifest V3.
+[Migration checklist][mv3-checklist]
+: Provides a quick checklist to help adapt your extension to Manifest V3.
 
-Manifest V3 represents one of the biggest shifts in the extensions platform
-since it launched a decade ago. Extensions using Manifest V3 will enjoy enhancements in
-security, privacy, and performance; they can also use more contemporary Open
-Web technologies adopted in Manifest V3, such as service workers and promises.
-Developers can update their extensions today to take advantage of these Manifest V3
-features; this will become mandatory as we phase out Manifest V2 in the future.
+## Start the conversion
 
-Manifest V3 is part of a shift in the philosophy behind how we approach
-end-user security and privacy. The pages in this section provide an overview of
-Manifest V3, the reasons behind it, and how to approach it:
+As of January 17, 2022 the Chrome Web Store has stopped accepting new Manifest V2 extensions. We strongly recommend to begin migrating your extensions to Manifest V3 as soon as possible; this will become mandatory as [Manifest V2 will be phasing out][mv2-sunset] in 2023.
```

Comment:
```suggestion
As of January 17, 2022 the Chrome Web Store has stopped accepting new Manifest V2 extensions. We strongly recommend migrating your extensions to Manifest V3 as soon as possible; this will become mandatory after [Manifest V2 is phased out][mv2-sunset] in 2023.
```